words/teams.txt

```diff
@@ -1,10 +1,27 @@
 alliance
+assembly
 band
+batch
+bunch
+club
+cluster
+combination
+conglomerate
 coven
 crew
+faction
+gathering
 group
+league
+lineup
+organization
+outfit
+pack
 party
 posse
 squad
 team
-troupe
\ No newline at end of file
+troop
+troupe
+trust
```

Comment:
Maybe add 'order' (like the Order of the Phoenix?)

Reply:
YES